changelogs/fragments/235-fix-net-put-issue.yaml

```diff
@@ -1,3 +0,0 @@
----
-bugfixes:
- - Remove temp file created when file already exist on destination when mode is 'text'.
```

Comment:
This didn't get caught in the PR, but I was hoping to fix this fragment so it mentioned that this is for the `net_put` module before running the changelog.

Reply:
Updated. Thanks